src/CodeGeneration/ApiGenerator/Views/_RequestParametersExtensions.Generated.cshtml

```diff
@@ -21,7 +21,7 @@ namespace Nest
 	{
 	@foreach (CsharpMethod method in Model.CsharpMethodsWithQueryStringInfo)
     {
-        if (!method.Url.Params.Any(p => p.Key.Contains("field") || p.Key.Contains("source_")))
+        if (!method.Url.Params.Any(p => p.Key.Contains("field") || p.Key.Contains("source_") || p.Key.Contains("routing")))
```

Comment:
These chained conditions always scare me as I have to read to work out what they mean.
Much prefer:

var nameOfCompositeCondition = ............................;
if (nameOfCompositeCondition) { ............ }

Reply:
👍